Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -388,8 +393,19 @@ private function _handleRequestAction($sActionTitle, $fAmount)
         $oTransactionHandler = $this->_getTransactionHandler();
         $aResult = $oTransactionHandler->processAction($this->oTransaction, $sActionTitle, $fAmount);
 
-        return $aResult["status"] === Transaction::STATE_SUCCESS ?
-            Helper::translate('wd_text_generic_success') : $aResult['message'];
+        $bSuccess = $aResult[self::KEY_STATUS] === Transaction::STATE_SUCCESS;
+
+        if (!$bSuccess) {
+            return [
```

Comment:
Let's create a static function that returns the "message" array.

Reply:
👍